crates/sloth_vm/src/sloth_std/misc.rs

```diff
@@ -0,0 +1,43 @@
+use super::NATIVE_LIBRARY;
+use crate::native::{self, NativeFunction, NativeFunctionResult};
+use crate::value::{Object, ObjectType, Primitive};
+use crate::VM;
+
+fn get_doc(vm: &mut VM, args: &[Primitive]) -> NativeFunctionResult {
+    let Some(Primitive::Object(ptr)) = args.get(0).cloned() else {
+        return Err(native::Error::InvalidArgument);
+    };
+
+    let object = vm
+        .objects()
+        .get(ptr as usize)
+        .ok_or(native::Error::InvalidArgument)?;
+
+    let ObjectType::NativeFunction(fnc) = &object.typ else {
+        return Err(native::Error::InvalidArgument);
+    };
+
+    let docs = NATIVE_LIBRARY
+        .get(fnc.name)
+        .ok_or(native::Error::InvalidArgument)?
+        .doc
+        .ok_or(native::Error::InvalidArgument)?
+        .to_string();
+
+    let object = Object::new(ObjectType::String(docs));
+    let ptr = vm.objects_mut().allocate(object);
+
+    Ok(Primitive::Object(ptr as u32))
+}
+
+pub const DOCS: NativeFunction = NativeFunction {
+    name: "docs",
+    function: get_doc,
+    arity: 1,
+    returns_value: true,
+    doc: Some(
+        "NativeFunction get_doc: \n\targs: name (str)\n\tdesc: Returns documentaiton on a \
+         function with name <str>\n\tExample: `var doc = get_doc('wait'); # Returns the \
+         documentation of the 'wait' function to doc`",
```

Comment:
Should be `docs` not `get_doc`

Reply:
```suggestion
        "NativeFunction get$doc: \n\targs: name (str)\n\tdesc: Returns documentaiton on a \
         function with name <str>\n\tExample: `var doc = get$doc('wait'); # Returns the \
         documentation of the 'wait' function to doc`",
```

Should fix it. Also I am doing all these commits from this github page itself so that is why every suggestion has its own commit...

Reply:
wait forgot I actually changed the name of `get$doc` to `docs`

crap.

Reply:
Fixed in my last commit